proto/user.proto

```diff
@@ -1,8 +1,8 @@
-message GoogleGroupGitHubUserMapping {
-    required string google_user_email = 1;
-    required string git_hub_user_id = 2;
+message UserMapping {
+    optional string google_user_email = 1;
+    optional string git_hub_user_id = 2;
```

Comment:
Why is this `git_hub` and not `github`?

Reply:
I will have a separate PR to change all `git_hub` to `github`